docs/ingesting_merfish_data/Ingesting_merfish_data_with_panpipes.md

```diff
@@ -0,0 +1,109 @@
+
+Next, in `spatial/ingestion` call `panpipes qc_spatial config` (you potentially need to activate the conda environment with `conda activate pipeline_env` first!). This will generate a `pipeline.log` and a `pipeline.yml` file.
+
+Customize the `pipeline.yml` or simply replace it with [the one we provide](pipeline.yml). Make sure to specify the correct path to the submission file. If you're using the [provided example yaml file](pipeline.yml), you potentially need to add the path of the conda environment in the yaml. 
+
+
+## Run Panpipes
+
+In `spatial/ingestion`, run `panpipes qc_spatial make full --local` to ingest your `MERFISH` datasets.
+
+After successfully running the pipeline, the `ingestion` folder looks as follows: 
+
+```
+ingestion
+├── data
+├── figures
+│   └── spatial
+│       ├── histograms.mouse_brain.png
+│       ├── spatial_spatial_n_genes_by_counts.mouse_brain.png
+│       ├── spatial_spatial_total_counts.mouse_brain.png
+│       ├── violin_obs_n_genes_by_counts_sample_id.mouse_brain.png
+│       ├── violin_obs_total_counts_sample_id.mouse_brain.png
+│       └── violin_var_total_counts.mouse_brain.png     
+├── logs
+│   ├── make_mudatas_mouse_brain.log  
+│   ├── qcplot.mouse_brain.log  
+│   └── spatialQC_mouse_brain.log
+├── qc.data # MuData with QC metrics 
+│   └──	mouse_brain_unfilt.h5mu
+├── tmp # MuData without QC metrics
+│   └──	mouse_brain_raw.h5mu
+├── pipeline.log
+├── pipeline.yml
+├── sample_file_qc_spatial.txt
+└── mouse_brain_cell_metadata.tsv # Metadata, i.e. .obs
+```
+
+In the `qc.data` folder, the final `MuData` object with computed QC metrics is stored. The `MuData` object without QC metrics is also available and stored in the `tmp` folder. The metadata of the final `Mudata` object is additionally extracted and saved as tsv file, `mouse_brain_cell_metadata.tsv`.
+Using the [provided example yaml file](pipeline.yml), the first rows and columns of the `mouse_brain_cell_metadata` tsv file look as follows: 
+
+|                  | spatial:fov | spatial:volume | spatial:min_x | spatial:max_x | spatial:min_y | spatial:max_y
+----------------------------------------|------------------|----------------------|----------------------|----------------------|------------------------|-----------------------
+110883424764611924400221639916314253469 |	     0         |  432.1414246586824	  |  151.53048381581905	 |  161.5960840769112   |     4264.619631592184	 |   4278.0332319401205	
+
+With the plots in `spatial/ingestion/figures/spatial` you can now decide on cutoffs for filtering. The plots include visualizations of the spatial embeddings, as well as violin plots, and histograms: 
+
+<img src="histograms.mouse_brain.png" alt="drawing" width="950"/>
```

Comment:
Write png as links to the github page, this way we don't have to store them on readthedocs. Also improves the build time.
The `?raw=true` at the end of each link is very important
```
<img src="https://github.com/DendrouLab/panpipes-tutorials/blob/sarah_readthedocs/docs/ingesting_merfish_data/histograms.mouse_brain.png?raw=true" alt="drawing1" width="950"/>
<p align="center">
<img src="https://github.com/DendrouLab/panpipes-tutorials/blob/sarah_readthedocs/docs/ingesting_merfish_data/spatial_spatial_n_genes_by_counts.mouse_brain.png?raw=true" alt="drawing2" width="200"/>
<img src="https://github.com/DendrouLab/panpipes-tutorials/blob/sarah_readthedocs/docs/ingesting_merfish_data/violin_obs_n_genes_by_counts_sample_id.mouse_brain.png?raw=true" alt="drawing3" width="200"/>
</p>
```

Reply:
Also please write more informative alt text